### DISCUSSION TITLE
post/pt-hack/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#15] [Announcements]
# post/pt-hack/

本文假设你已经对什么是 BT(BitTorrent) 以及 PT(Private Tracker) 有基本了解.  0x00 前言(回忆&废话)遥想十几年前, 我还在小学低年级的时候家里没有宽带, 只能用 3G 无线上网卡上网, 网速不超过 5Mbps. 在线看优酷都只能看低分辨率版本的. 那时候也用 BT 种子和硕鼠 flv 下载器之类的方法囤积过一些视频 (那时候下载的电影都还是 rm

https://blog.lyc8503.site/post/pt-hack/

**Comment by LimLLL** [comment]
佬写的真好，因为家宽数据不佳，求不到大站💊，导致一直属于被pt玩，很在意ratio。意外看到佬的文章，以及看了看佬文章里的两个介绍bt的链接，然后用nodejs+react重写了个全自动刷假流做假种的小玩意儿😂虽然还是没人发啥大站，但是起码无ratio焦虑了。(佬别看不起我的行为😂

**Reply by lyc8503** [reply]
能自己写点小程序挺好的😶 ~~但作弊肯定还是不提倡的, 当心被封号然后邀请人连坐.~~

ratio 也不用太焦虑, 要不真的就是被 PT 反向 pua, 家宽调好端口映射随缘挂着即可, 没必要一直盯着.

**Reply by yuanhuat** [reply]
可以给我一份吗

**Reply by lyc8503** [reply]
@yuanhuat 我已经在文末提供了 GitHub 仓库链接, 其中包含完整的代码. 这只是一个实验性质的代码, 不是现成的作弊工具. 如果你不会使用 GitHub 或连文章都没耐心看完, 即使你跑起来了, 也很可能因为错误使用被 ban.

**Comment by cdowen** [comment]
最近也发现bt协议并没有详细的流量统计协议，在网上找资料的时候发现了这篇。写的真好，完全解惑了。

**Comment by arielherself** [comment]
感谢大佬的分析，我之前不太了解 tracker 的工作原理，看了这篇博客又在网上搜索之后基本上懂了，写了一个免费下载的 web 服务，可以缓存 peer 列表。如果有人感兴趣可以来用 https://tracker.submy.org
